### PR TITLE
Rtps updates

### PR DESCRIPTION
**Pull request by hamishwillee**
@LorenzMeier 
This PR tidies up the RTK GPS docs as requested.
1. The ublox config doc is moved from /tutorials (which it was not) to /advanced (the next best thing to "background") and renames it to "RTK GPS (Background)
2. Some general/minor tidy up and cross linking.
3. I did not do anything with the [DJI Flamewheel build log](https://dev.px4.io/en/airframes_multicopter/dji_flamewheel_450.html) - that is a good build reference that is linked from the user guide.

There is one further improvement I could do, which will be linked by comment inline.

### CODE REVIEW

File: en/advanced/rtk_gps.md [hunk 1]
```diff
@@ -1,54 +1,60 @@
...
 
 QGroundControl configures the RTK base station to output the following RTCM3.2 frames, each with 1 Hz:
+
 - **1005** - Station coordinates XYZ for antenna reference point (Base position).
 - **1077** - Full GPS pseudo-ranges, carrier phases, Doppler and signal strength (high resolution).
 - **1087** - Full GLONASS pseudo-ranges, carrier phases, Doppler and signal strength (high resolution).
 
 
-### Uplink Datarate
+## Uplink datarate
```
Comment (hamishwillee) [inline en/advanced/rtk_gps.md]: I could improve this but I need to clarify MAVLink2 vs MAVLink1. 

This says "The length of each MAVLink message is 182 bytes," -  Does the 182 bytes have any relevance to MAVLink2 as well - ie is it a "maximum" length or is this purely a mavlink1 thing.

My interpretation as it is written now is that this is a maximum size for both MAVLink1 and 2. However on MAVLInk 1 the message is always sent at this size while Mavlink2 can send a message that varies in size based on the content.

Comment (LorenzMeier) [reply en/advanced/rtk_gps.md]: Your interpretation is correct down to the last word.

**Comment by hamishwillee** [comment]
Thanks. I made minor changes to the uplink section. Feel free to merge if you're happy.

**Comment by hamishwillee** [comment]
Merging. This makes sense and changes only organisation, not content.

**Pull request merged**